tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/identifiers/IdentifierTest.java

```diff
@@ -64,6 +64,7 @@ public void testInvalidImportOnMultipleFiles() {
         validateError(result, index++, "redeclared symbol 'error'", 19, 18);
         validateError(result, index++, "redeclared symbol 'error'", 30, 11);
         validateError(result, index++, "redeclared symbol 'error'", 37, 73);
+        validateError(result, index++, "redeclared symbol 'error'", 39, 39);
```

Comment:
Please create an issue to improve this error. Ideally we should give a more descriptive error.